content/en/monitors/create/types/ci_pipelines.md

```diff
@@ -0,0 +1,119 @@
+4. Configure the alerting grouping strategy (optional):
+    * If the query has a `group by`, multi alerts apply the alert to each source according to your group parameters. An alerting event is generated for each group that meets the set conditions. For example, you could group a query by `@ci.pipeline.name` to receive a separate alert for each CI Pipeline name when the number of errors is high.
+
+{{< img src="monitors/monitor_types/ci_pipelines/define-the-search-query.png" alt="Define the search query" style="width:80%;" >}}
+
+#### Using Formulas & Functions
+
+You can create CI Pipeline monitors based on Formulas & Functions. This is useful if you want to trigger alerts for monitors based on the error rate for some pipeline.
+
+In the following example, we're creating an error rate monitor using the formula that calculates the ratio of `Failed CI Pipelines` / `Total CI Pipelines` grouped by `ci.pipeline.name`.
+
+{{< img src="monitors/monitor_types/ci_pipelines/define-the-search-query-fnf.png" alt="Define the search query" style="width:80%;" >}}
+
+<div class="alert alert-info"><strong>Note</strong>: Only up to 2 queries can be used to build the evaluation formula per monitor.</div>
+
+### Set alert conditions
+
+* Trigger when the metric is `above`, `above or equal to`, `below`, or `below or equal to`
+* the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, etc. or `custom` to set a value between `1 minute` and `2 days`.
+* Alert threshold `<NUMBER>`
+* Warning threshold `<NUMBER>`
+
+#### Advanced alert conditions
+
+For detailed instructions on the advanced alert options (evaluation delay, etc.), see the [Monitor configuration][3] page.
+
+### Notifications
+
+For detailed instructions on the **Say what's happening** and **Notify your team** sections, see the [Notifications][4] page.
+
+#### Enrich the monitor message with event attributes
+
+You can use all the attributes stored in a CI Pipeline event in your notification message using the `{{cipipeline.attributes.<attribute>}}` syntax. You can use all the CI Pipeline facets to build your custom notification message.
+
+In the following example, we're configuring a notification message using the Git repository URL stored in the CI Pipeline event attributes.
+
+```text
+{{#is_alert}}
+This is a sample notification message for the repository {{cipipeline.attributes.git.repository_url}}
+{{/is_alert}}
+```
+
+You can mix CI Pipeline event attributes with the variable populated through the facet used to group the alerts.
+
+In the following example, we're configuring a notification message using the Git repository URL stored in the CI Pipeline event attributes and using the variable populated using the facet to group the alerts.
+
+```text
+{{#is_alert}}
+This is a sample notification message for the repository {{cipipeline.attributes.git.repository_url}} in the {{[@ci.pipeline.name].name}}
+{{/is_alert}}
+```
+
```

Comment:
All this is already included in the link of the previous section (https://docs.datadoghq.com/monitors/notify/variables/?tab=is_alert#matching-attributetag-variables)

No other monitor type-specific page explains this, so I'd follow the same approach.

```suggestion
```